### ISSUE TITLE
Change after 4/18 now downloading 1080p DASH instead of native format?

### ISSUE THREAD

**Issue by CollinChaffin** [rg3/youtube-dl#5825]
Now, all my videos after 4.18 began re-downloading (hundreds gigs) because someone decided that ffmpeg 1080p dash all the sudden as default was a good idea without warning.  Where was this change made to stop pulling the highest **NATIVE ORIGINAL non-dash no conversion or merge needed** which in <most> cases was 720p?

This is absurd to make that change which when pointed at existing downloads made all old ones look invisible since they did no longer adhere to this new highest format of the merged DASH.  I don't want the merged DASH I don't want any merging of audio and video, I was (as I am guessing many others were) happy with the native, original uploaded 720p without any fake upscaling.  It was also a LOT faster since no ffmpeg was needed at all to simply pull down the single MP4.

So, can someone help with a reasonable workaround - putting in 720p as the highest allowed isn't really a solution is it?  When I use the no dash manifest for some reason the unmerged 1080p still downloads.

Any help greatly appreciated!

**Comment by dstftw** [comment]
Calm down and read the last paragraph of [format selection section](https://github.com/rg3/youtube-dl#format-selection).

**Comment by jaimeMF** [comment]
I'm going to also answer to your comments in https://github.com/rg3/youtube-dl/commit/feccf29c876869f44a9c983977371073b9801a51 here:

>  I honestly missed it but judging from other folks comments I know I'm not alone I still think a new cmdline for this would been better idea I do see value to new behavior 

We have actually got a few requests for making downloading the DASH formats the default behaviour, most people expect youtube-dl to download the best quality without any additional parameters (that's what we do for all sites). Making it the default is also important because the feature for merging formats is not easily discoverable, unless you were told it existed you would think it wasn't implemented.

>  I don't want the merged DASH I don't want any merging of audio and video, I was (as I am guessing many others were) happy with the native, original uploaded 720p without any fake upscaling

For every case of an upscaled video there will be a video whose native resolution won't be served as a single file. For example:
* native 1080p videos, they are quite common.
* As far as I know 60fps videos are only available as DASH, and some people really appreciate having that framerate.
* all those 4k video.

In conclusion, we make the switch because it would allow most users to get the best version by default and people who don't want them can just add `-f best` to their config file. I'm pretty sure that there were a lot of people that had `-f bestvideo+bestaudio/best` in their configuration.
